lib/flags.rb

```diff
@@ -0,0 +1,819 @@
+class CustomWizard::Flag
+    def initialize(id, name, value)
```

Comment:
@merefield Just a small note about formatting. We need to follow Discourse tab policy here, i.e. single tab indentation.

Reply:
This is resolved for this file pending commit